### PR TITLE
Foundations: Add Node Installation instructions as a lesson before JavaScript Fundamentals Part 4

### PR DESCRIPTION
**Pull request by programmurr**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Moves the Node Installation instructions out to be their own lesson to be placed before JavaScript Fundamentals Part 4. At the moment they are among the instructions for cloning the JavaScript Exercise repo and installing Jest. Breaking them out into their own lesson may ease this friction and reduce the number of questions relating to it in the Discord.

#### 2. Related Issue

This is related to issue [#23248](https://github.com/TheOdinProject/curriculum/issues/23248) but does not close it. If this change is approved and merged, then the change needs to be made in the TOP website repo and the JavaScript Exercises [README](https://github.com/TheOdinProject/javascript-exercises#readme) needs updated to reflect the change.

### CODE REVIEW

**Review by fortypercenttitanium — APPROVED** [review]
Nice work, man! I'd like to get another set of eyes before we merge. Thanks for your patience.

File: foundations/javascript_basics/installing_nodejs.md
Comment (twalton83) [inline foundations/javascript_basics/installing_nodejs.md:112]: I’d like to emphasize that they MUST use the LTS to avoid incompatibilities. Otherwise looks good.

**Review by twalton83 — APPROVED** [review]

File: foundations/javascript_basics/installing_nodejs.md
Comment (fortypercenttitanium) [reply foundations/javascript_basics/installing_nodejs.md:112]: Great idea, and maybe use that as a way to briefly mention *what* LTS means in this case.

Comment (programmurr) [reply foundations/javascript_basics/installing_nodejs.md]: Added in the new commit

File: foundations/javascript_basics/installing_nodejs.md
Comment (fortypercenttitanium) [inline foundations/javascript_basics/installing_nodejs.md:112]: > it's initial release

change to "its" (no apostrophe)

Comment (fortypercenttitanium) [inline foundations/javascript_basics/installing_nodejs.md:112]: I'd get rid of the bit about even and odd numbering. It's extraneous for the purposes here. The rest is great, well done!

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]
Just two little things, then LGTM!

File: foundations/javascript_basics/installing_nodejs.md
Comment (programmurr) [reply foundations/javascript_basics/installing_nodejs.md]: Fixed in new commit

Comment (programmurr) [reply foundations/javascript_basics/installing_nodejs.md]: Fixed in new commit. Thank you :)

**Review by fortypercenttitanium — APPROVED** [review]

**Pull request merged**
